exchanges/anx/anx_test.go

```diff
@@ -283,3 +283,17 @@ func TestCancelExchangeOrder(t *testing.T) {
 		t.Errorf("Test Failed - ANX CancelOrder() error: %s", err)
 	}
 }
+
+func TestGetAccountInfo(t *testing.T) {
```

Comment:
I'm not understanding this test. It looks like both do the same thing

Reply:
I dont want to skip because it increases coverage also if you do not get an error returned usually means the errors are not getting captured, there are a few I had to specifically write in the SendAuthHTTPRequest funcs which was found by using this method.